### ISSUE TITLE
GUNDAM BREAKER 3 BREAK EDITION [PCSH10041]

### ISSUE THREAD

**Issue by holybrunneis** [Vita3K/compatibility#509]
# App summary
- App name: GUNDAM BREAKER 3 BREAK EDITION
- App serial: PCSH10041
- App version: 1.00

# Vita3K summary
- Version: v0.1.3
- Build number: 2237
- Commit hash: https://github.com/vita3k/vita3k/commit/[commit] 
- CPU backend: Dynarmic             

                          

# Test environment summary
- Tested by: brunneis
- OS: Windows 10
- CPU: Intel Core i7-9750H
- GPU: Nvidia GTX 1660 Ti
- RAM: 16 GB

# Issues

Instant crash after launch.

# Screenshots
![image_2021-06-24_045316](https://user-images.githubusercontent.com/65845653/123242898-16f9dc80-d4a8-11eb-8618-3b094a244b31.png)

# Log
[vita3k.log](https://github.com/Vita3K/compatibility/files/6708188/vita3k.log)

# Recommended labels

- Nothing

**Comment by hobyst** [comment]
In the log, the game reports a lot of missing files before crashing, which likely means the game dump the user used with the emulator is corrupt or incomplete.

**Comment by Zangetsu38** [comment]
> In the log, the game reports a lot of missing files before crashing, which likely means the game dump the user used with the emulator is corrupt or incomplete.

no,att all, file missing is totaly normal 🤷‍♂️ , this game crash on boot that all, i suspect ngs

**Comment by othmanead** [comment]
# Vita3K summary
- Version: v0.1.5
- Build number: 2723
- Commit hash: https://github.com/vita3k/vita3k/commit/c32ac263
- CPU backend: Dynamic  

# Test environment summary
- Tested by: othmanead
- OS: Windows 11
- CPU: Intel i5-10300H
- GPU: RTX 3060 mobile
- RAM: 16 GB


# Issue: 
Game goes ingame but besides the hud everything is black

# Screenshots:
![image](https://user-images.githubusercontent.com/106798110/177504357-a6234b0f-585d-413f-a770-7e08df216153.png)



# Log




[vita3k.log](https://github.com/Vita3K/compatibility/files/9052974/vita3k.log)



# Recommended labels
- Ingame -
- Graphics bug

**Comment by github-user1112** [comment]
# Vita3K summary
Version: v0.2.1
Build number: 3967
Commit hash: https://github.com/Vita3K/commit/9916d86
GPU backend: Vulkan

# Test environment summary
Tested by: gtihub-user
OS: Linux Mint 22.3
CPU: AMD Ryzen 5 5600
GPU: Radeon Rx 6600
RAM: 16 GB

# Issues
Colours and graphics are visible but all messed up now


<img width="960" height="1042" alt="Image" src="https://github.com/user-attachments/assets/805173c0-08ee-4772-a50e-8b5f0807a8bd" />

<img width="960" height="1042" alt="Image" src="https://github.com/user-attachments/assets/0776d59f-6578-495c-9c9b-e4dc854e28c6" />
